src/pkg/services/m365/api/lists_test.go

```diff
@@ -68,6 +69,389 @@ func (suite *ListsUnitSuite) TestBytesToListable() {
 	}
 }
 
+func (suite *ListsUnitSuite) TestColumnDefinitionValidationSetWhenProvided() {
```

Comment:
For test naming, code standard is to follow golang's testable example naming schemes.  see: https://go.dev/blog/examples#example-function-names

In short, it goes something like `TestStructName_FuncName_subtestExplanation`.

so in this case you're testing:
```suggestion
func (suite *ListsUnitSuite) TestColumnDefinitionable_GetValidation() {
```